packages/console/src/components/Sidebar/components/Section/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { ReactNode } from 'react';
+
+import * as styles from './index.module.scss';
```

Comment:
do we support `import styles from './index.module.scss';`

Reply:
I think there's a lint rule that prefers `import * as styles` instead of `import styles`. Seen it once.

Reply:
parcel requires this for treeshaking

Reply:
Interesting... Searched and found this:

<img width="730" alt="image" src="https://user-images.githubusercontent.com/12833674/156105830-ed70b562-f681-42a1-a27b-d4db1a5f9f71.png">
https://parceljs.org/languages/css/#tree-shaking